### PR TITLE
Convert html to react -> header.jsx and Homepage.jsx

### PR DESCRIPTION
**Pull request by AkankshShigwan**

### CODE REVIEW

File: ui/src/components/Header.jsx
Comment (prasadhonrao) [inline ui/src/components/Header.jsx:1]: Import sequnce should be like following

1. react specific
2. Third part like font awesome
3. Project specific files like boostrap.css

File: ui/src/components/bootcamps.js
Comment (prasadhonrao) [inline ui/src/components/bootcamps.js:1]: Filename should be Bootcamps.jsx

File: ui/src/components/login.js
Comment (prasadhonrao) [inline ui/src/components/login.js:1]: Filename should be Login.jsx

File: ui/src/components/register.jsx
Comment (prasadhonrao) [inline ui/src/components/register.jsx:1]: Do we need a separate component for Register page. Are we using this anywhere else in the project?

**Review by prasadhonrao — CHANGES_REQUESTED** [review]
Comments added in individual files.

File: ui/src/components/register.jsx
Comment (AkankshShigwan) [reply ui/src/components/register.jsx:1]: not we will have only one registration page

**Comment by prasadhonrao** [comment]
Close by #50

**Pull request closed without merging**